### PR TITLE
add: existing environment when modifying endpoint or querying endpoint detail

### PR DESCRIPTION
**Pull request by lizable**
This PR loads saved environment variable of endpoint on update and routing detail for better usability.
By this feature, user can track what they input when executing a service.

### Screenshot(s)
Screenshot 2024-07-08 at 10 52 22 PM
Screenshot 2024-07-08 at 10 52 40 PM

**Checklist:** (if applicable)

- [ ] Mention to the original issue
- [ ] Documentation
- [ ] Minium required manager version
- [ ] Specific setting for review (eg., KB link, endpoint or how to setup)
- [ ] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

**Comment by lizable** [comment]
* **#2525** <a href="https://app.graphite.dev/github/pr/lablup/backend.ai-webui/2525?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @lizable and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: react/src/components/ServiceLauncherPageContent.tsx [hunk 1]
```diff
@@ -634,6 +638,9 @@ const ServiceLauncherPageContent: React.FC<ServiceLauncherPageContentProps> = ({
         modelMountDestination: endpoint?.model_mount_destination,
         modelDefinitionPath: endpoint?.model_definition_path,
         runtimeVariant: endpoint?.runtime_variant?.name,
+        envvars: Object.entries(JSON.parse(endpoint?.environ)).map(
+          ([variable, value]) => ({ variable, value }),
+        ),
```
Comment (agatha197) [inline react/src/components/ServiceLauncherPageContent.tsx]:
```suggestion
        envvars: _.map(
          JSON.parse(endpoint?.environ || '{}'),
          (value, variable) => ({ variable, value }),
        ),
```

File: react/src/pages/EndpointDetailPage.tsx [hunk 2]
```diff
@@ -380,6 +381,18 @@ const EndpointDetailPage: React.FC<EndpointDetailPageProps> = () => {
     });
   }
 
+  items.push({
+    label: t('session.launcher.EnvironmentVariable'),
+    children: (
+      <Typography.Text style={{ fontFamily: 'monospace' }}>
+        {_.isEmpty(JSON.parse(endpoint?.environ)) ? 'None' : endpoint?.environ}
```
Comment (agatha197) [inline react/src/pages/EndpointDetailPage.tsx]:
```suggestion
        {_.isEmpty(JSON.parse(endpoint?.environ)) ? '-' : endpoint?.environ}
```

**Review by agatha197 — CHANGES_REQUESTED** [review]

**Review by agatha197 — APPROVED** [review]
LGTM

**Pull request merged**